balancer/pickfirst/pickfirstleaf/pickfirstleaf.go

```diff
@@ -85,7 +89,8 @@ func (pickfirstBuilder) Build(cc balancer.ClientConn, _ balancer.BuildOptions) b
 		cc:                    cc,
 		addressList:           addressList{},
```

Comment:
You should be able to just leave this (and `mu`) out, since you are using their default constructors.